Fix hard-coding of /usr/bin/perl by using Perl available through $PATH

FastQC has moved to a Python launcher as of version 0.13.0, but a legacy Perl
launch script is still available.

author: Georgios Kafanas (UL HPC)
diff -ruN FastQC.orig/fastqc.pl FastQC/fastqc.pl
--- FastQC.orig/fastqc.pl	2026-10-02 14:50:54.000000000 +0200
+++ FastQC/fastqc.pl	2026-10-14 22:11:50.445281111 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 use warnings;
 use strict;
 use FindBin qw($RealBin);
